formats/CLIENT_COMMANDS.md

```diff
@@ -0,0 +1,75 @@
+# Redis Client Library Command Specification
+
+Redis client libraries provide helper functions for communicating with Redis. This specification exists to describe the format of the file used by redis clients, to communicate their function usage with Redis.
+
+## Implementation
+
+To implement this specification, a file named *usage.json* must be created at the root of the client repository. *usage.json* is a json formatted file containing links to generated documentation, and optionally examples.
+
+The following is the basic example of the *usage.json* specification.
+
+::
+    {
+      "name": "redis-py",
+      "language": "python",
+      "metadata": {
+        "repository": "https://github.com/redis/redis-py",
+        "issues_url": "https://github.com/redis/redis-py/issues"
+      },
+      "commands": {
+        "SET": {
+            "uri": "https://redis.readthedocs.io/en/latest/commands.html#redis.commands.core.CoreCommands.set",
```

Comment:
Given that we have more than one connection - IMHO we should have a list of URIs. This would solve the Java use case.  WDYT @sazzad16 @leibale